python/ml4ir/applications/classification/tests/test_classification_model.py

```diff
@@ -32,7 +33,8 @@ def test_csv_loss_metrics(self):
                         msg=f"Top5 Categorical_accuracy not in expected range."
                             f" Expected: {expected_acc} ± {tol}, Found: {self.metrics_dict[_metric]}")
 
-    def test_group_metrics_df(self):
+    @pytest.mark.skip(reason="Pending on planning for classification model evaluation")
```

Comment:
if we set the argparser arg to True, or when we have a way to control it, can this test pass?

Reply:
Yes